src/hibpcli/password.py

```diff
@@ -23,8 +23,7 @@ def is_leaked(self):
             # one entry consists of the rest of the hash and count of
             # leak, separated by a colon
             # cut off string - information after the hash is of no interest
-            partial_hash_list = [line[:35] for line in result.splitlines()]
-            return second_hash_part in partial_hash_list
+            return any(second_hash_part == line[:35] for line in result.splitlines())
```

Comment:
As discussed on Signal, could you change the line to `return second_hash_part in (line[:35] for line in result.splitlines())` ?

Reply:
```suggestion
            return second_hash_part in (line[:35] for line in result.splitlines())
```